src/providers/NetworkProvider.tsx

```diff
@@ -63,6 +63,7 @@ export const NetworkProvider: React.FC = ({ children }) => {
     const config: Subscriptions = {
       address: setAccount,
       wallet: selectWallet,
+      network: networkId => setNetwork(NETWORKS[networkId]?.name),
```

Comment:
@wraeth-eth i like your fix here. Wondering if you think a hard page reload might be worthwhile instead of this though 🤔 . Just to be safe.

Reply:
definitely on board with a hard page refresh @aeolianeth - just smooths over any other bumps that might occur. Where is the best place to do that - in this function, or somewhere else?

Reply:
Have updated - its not the best UX, but it works. I am uncertain if user will be swapping networks often while on JB too